src/test/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/VersionDifferencesTest.kt

```diff
@@ -11,17 +11,44 @@ import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.service.ConditionCh
 import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.service.ConditionChangeType.REPLACED
 import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.service.ConditionChangeType.TEXT_CHANGE
 
-class RealLicencePolicyServiceTest() {
+class VersionDifferencesTest() {
 
   private val licencePolicyService = LicencePolicyService(
     listOf(
+      ClassPathResource("/policy_conditions/policyV1.json").file,
       ClassPathResource("/policy_conditions/policyV2.json").file,
       ClassPathResource("/policy_conditions/policyV2.1.json").file
     ).map { policy -> jacksonObjectMapper().readValue(policy) }
   )
 
   @Test
-  fun `Current policy version is returned`() {
+  fun `1_0 to 2_0`() {
+    val currentPolicy = licencePolicyService.policyByVersion("2.0")
+    val previousPolicy = licencePolicyService.policyByVersion("1.0")
+
+    val replacements = getSuggestedReplacements(previousPolicy, currentPolicy)
+
+    val diffs =
+      conditionChanges(previousPolicy.additionalConditions.ap, currentPolicy.additionalConditions.ap, replacements)
+
+    showDiffs(diffs)
+  }
+
+  @Test
+  fun `1_0 to 2_1`() {
+    val currentPolicy = licencePolicyService.policyByVersion("2.1")
+    val previousPolicy = licencePolicyService.policyByVersion("1.0")
+
+    val replacements = getSuggestedReplacements(previousPolicy, currentPolicy)
+
+    val diffs =
+      conditionChanges(previousPolicy.additionalConditions.ap, currentPolicy.additionalConditions.ap, replacements)
+
+    showDiffs(diffs)
+  }
```

Comment:
There are quite a lot of records in the DB that are at version 1.0 that could be migrated to version 2.1. 

I think it doesn't matter as there are no backward incompatible changes so the diffs is empty